QUESTIONS.md

```diff
@@ -1,14 +1,83 @@
-partial refunds ()
+Questions:
+- can we combine grants structs
+- can we get rid of grant types?
+- can we reduce the # of grant status?
+- can we reduce # of roles (grantor / grantee)
+- can we combine / reduce grant state
+- do we want to store grant recipients as array or mapping
+  - do we need to iterate over this in the contract logic
+- can we remove "request payment"
+  - are we assuming that grant managers have robust offchain comm with grantees
+  - what if everyone is a grant manager?
+- can we remove Payments[] by having a "requesting" global state
+  - grant managers approve / vote on the requested funds
+  - if expires then can propose again
+  - possibly use timestamp to avoid explicit states
+- can we remove multiple grant managers / voting / m-of-n and replace w/ msig?
+- use donor / grantee / manager
+- maybe use a getter for total funded (diff for ERC20 vs ETH)
+
+Design Question:
+- Currently, the grant manager is bound by the initial allocation %s, they can
```

Comment:
@JFickel thoughts on this one

Reply:
I don't think grant managers should be able to re-allocate. Seems like a security hole and unexpected behavior for funders